script.js

```diff
@@ -1,4 +1,43 @@
 import { Producto } from "./Producto.js";
+import PackAlternativo from "./packAlternativo.js";
+
+
+
+
+document.getElementById("contact-form").addEventListener("submit", function(event) {
+    // prevenir que el formulario realice la acción por defecto (refrescar la página x ej)
+    event.preventDefault();
+
+    //agarrar los valores de los campos del formulario
+    const nombre = document.getElementById("first-name").value;
+    const apellido = document.getElementById("last-name").value;
+    const empresa = document.getElementById("company").value;
+    const mensaje = document.getElementById("message").value;
+
+    //objeto con los datos del formulario
+    const datosContacto = {
+        nombre: nombre,
+        apellido: apellido,
+        empresa: empresa,
+        mensaje: mensaje
+    };
+
+    // se almacena los datos en localStorage como JSON
+    localStorage.setItem("contacto", JSON.stringify(datosContacto));
```

Comment:
No hay validación de campos vacíos antes de guardar los datos. Esto puede resultar en entradas no deseadas.
```suggestion
if (!nombre || !apellido || !empresa || !mensaje) {
  Swal.fire({
    title: "Error",
    text: "Por favor, completa todos los campos.",
    icon: "warning",
    confirmButtonText: "Aceptar",
  });
  return;
}
```